### PR TITLE
🔨(backend) migration docker compose

### PR DESCRIPTION
**Pull request by rlecellier**
docker-compose command doesn't work anymore.
we need to switch to docker compose

```sh
git grep -rl docker-compose . | xargs sed -i 's/docker-compose/docker compose/g'
git grep -rl "docker compose file" . | xargs sed -i 's/docker compose file/docker-compose file/g'
git grep -rl "../docker compose.yml" . | xargs sed -i 's/docker compose.yml/docker-compose.yml/g'
```

migration: 
```sh
$ make down
$ docker-compose down
$ make boostrap
$ make stop && make run # postgresql seems to take time to initialize and need a reboot
$ make migrate
$ make demo-site
```

### CODE REVIEW

File: bin/_config.sh [hunk 1]
```diff
@@ -9,9 +9,9 @@ COMPOSE_PROJECT="richie"
 # By default, all commands are run on Postgresql.
 # They can be run on Mysql by setting an environment variable: `DB_ENGINE=mysql`
 if [[ "${DB_ENGINE}" == "mysql" ]]; then
-    COMPOSE_FILE="${REPO_DIR}/docker/compose/development/mysql/docker-compose.yml"
+    COMPOSE_FILE="${REPO_DIR}/docker/compose/development/mysql/docker compose.yml"
```
Comment (kernicPanel) [inline bin/_config.sh:12]:
```suggestion
    COMPOSE_FILE="${REPO_DIR}/docker/compose/development/mysql/docker-compose.yml"
```

File: bin/_config.sh [hunk 2]
```diff
@@ -9,9 +9,9 @@ COMPOSE_PROJECT="richie"
...
 else
-    COMPOSE_FILE="${REPO_DIR}/docker-compose.yml"
+    COMPOSE_FILE="${REPO_DIR}/docker compose.yml"
```
Comment (kernicPanel) [inline bin/_config.sh:14]:
```suggestion
    COMPOSE_FILE="${REPO_DIR}/docker-compose.yml"
```

File: cookiecutter/{{cookiecutter.organization}}-richie-site-factory/.dockerignore [hunk 3]
```diff
@@ -11,7 +11,7 @@ venv
 **/.DS_Store
 
 # Docker
-docker-compose.*
+docker compose.*
```
Comment (kernicPanel) [inline cookiecutter/{{cookiecutter.organization}}-richie-site-factory/.dockerignore:14]:
```suggestion
docker-compose.*
```

File: website/versioned_docs/version-1.13/quick-start.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ Login/password are `admin`/`admin`. The database is regularly flushed.
 `Richie` is a **container-native application** but can also be installed
 [on your machine](native-installation.md).
 
-For development, the project is defined using a [docker-compose file](../docker-compose.yml) and
+For development, the project is defined using a [docker compose file](../docker compose.yml) and
```
Comment (kernicPanel) [inline website/versioned_docs/version-1.13/quick-start.md:16]:
```suggestion
For development, the project is defined using a [docker compose file](../docker-compose.yml) and
```

File: website/versioned_docs/version-1.14/quick-start.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Login/password are `admin`/`admin`. The database is regularly flushed.
 `Richie` is a **container-native application** but can also be installed
 [on your machine](native-installation.md).
 
-For development, the project is defined using a [docker-compose file](../docker-compose.yml) and
+For development, the project is defined using a [docker compose file](../docker compose.yml) and
```
Comment (kernicPanel) [inline website/versioned_docs/version-1.14/quick-start.md:16]:
```suggestion
For development, the project is defined using a [docker compose file](../docker-compose.yml) and
```

**Review by kernicPanel — COMMENTED** [review]
I don't think that the versionned doc should be edited.

**Comment by jbpenrath** [comment]
You should also remove the property `version` into the docker-compose files (at the root of the repo and within the cookiecutter project)

File: cookiecutter/{{cookiecutter.organization}}-richie-site-factory/docker-compose.yml
Comment (jbpenrath) [inline cookiecutter/{{cookiecutter.organization}}-richie-site-factory/docker-compose.yml:1]: Do not forgot to update network name also here

File: cookiecutter/{{cookiecutter.organization}}-richie-site-factory/.dockerignore (hunk 3 above)
Comment (kernicPanel) [reply cookiecutter/{{cookiecutter.organization}}-richie-site-factory/.dockerignore:14]: Sorry, but we have to keep `docker-compose.*` here, as it is a file.

File: docs/installation.md [hunk 6]
```diff
@@ -37,8 +37,8 @@ laptop:
 $ docker -v
   Docker version 20.10.12, build e91ed57
 
-$ docker-compose --version
-  docker-compose version 1.29.2, build 5becea4c
+$ docker compose --version
+  docker compose version 1.29.2, build 5becea4c
```
Comment (kernicPanel) [inline docs/installation.md:41]:
```suggestion
$ docker compose version
Docker Compose version v2.25.0
```

**Review by kernicPanel — COMMENTED** [review]
I still believe we shouldn't update versioned docs. WDYT ?

File: website/versioned_docs/version-1.12/contributing.md
Comment (jbpenrath) [inline website/versioned_docs/version-1.12/contributing.md:1]: Agreed with @kernicPanel, the versioned doc should not be updated.

**Review by jbpenrath — APPROVED** [review]
LGTM ✅

**Pull request merged**